src/main/java/ch/jalu/configme/configurationdata/CommentsConfiguration.java

```diff
@@ -41,7 +41,11 @@ public CommentsConfiguration(@NotNull Map<String, List<String>> comments) {
      * @param commentLines the comment lines to set for the path
      */
     public void setComment(@NotNull String path, @NotNull String... commentLines) {
-        comments.put(path, Collections.unmodifiableList(Arrays.asList(commentLines)));
+        List<String> replaced = comments.put(path, Collections.unmodifiableList(Arrays.asList(commentLines)));
+        
+        if(replaced != null) {
+            throw new IllegalStateException("Comment lines already exists for the path " + path);
```

Comment:
I might fiddle with the exception text sometime later (I'm a bit pressed for time today) but what would you think of this for now?
```suggestion
            throw new IllegalStateException("Comments were already registered for the path '" + path + "'");
```

This is a discussion btw, maybe you have a good idea. Enclosing the path in single quotes is a habit I took over from my former employer—the idea being that it makes it clearer to the user what part of the texts are "dynamic" strings. Also makes the text a bit clearer in case someone uses the root path `""`